R/latex.R

```diff
@@ -234,18 +234,24 @@ restore_block2 = function(x, global = FALSE) {
   # fenced div (\begin) is used
   if (length(grep(sprintf('^\\\\(BeginKnitrBlock|begin)\\{(%s)\\}', paste(all_math_env, collapse = '|')), x)) &&
       length(grep('^\\s*\\\\newtheorem\\{theorem\\}', head(x, i))) == 0) {
+    theorem_label = vapply(
+      lapply(theorem_abbr, label_prefix),
+      function(fun) fun(), character(1), USE.NAMES = FALSE)
```

Comment:
Is it equivalent to this?

```suggestion
    theorem_label = vapply(theorem_abbr, function(a) {
      label_prefix(a)()
    }, character(1), USE.NAMES = FALSE)
```

Reply:
Oh yes! Thanks - don't know how I missed that. Sometimes I do too complicated 😅